sentry/src/test/java/io/sentry/SentryEnvelopeTest.kt

```diff
@@ -137,4 +135,25 @@ abcdefghij""".toInputStream()
         assertEquals(10, secondItem.header.length)
         assertEquals(10, secondItem.data.size)
     }
+
+    @Test
+    fun `deserializes an user feedback`() {
```

Comment:
`l`: I think the test name could reflect the envelope is coming from dart.

Reply:
same as https://github.com/getsentry/sentry-java/pull/1611#discussion_r672971707